e2e/runner.go

```diff
@@ -98,8 +98,7 @@ func runTest(t *testing.T, testCase *TestCase) {
 
 	if testCase.CIUser == LoggedInCIUser {
 		if os.Getenv(envvarWithE2EUserToken) == "" {
-			fmt.Println("You are trying to test LoggedIn behavior, you need to set " + envvarWithE2EUserToken + " envvar.")
-			t.FailNow()
+			require.FailNow(t, "You are trying to test LoggedIn behavior, you need to set "+envvarWithE2EUserToken+" envvar.")
```

Comment:
I noticed the wording of `LoggedIn behavior` to be odd. Maybe change this to 

```suggestion
			require.FailNow(t, "You are trying to test behavior that requires a connection to CI Sense. Please set "+envvarWithE2EUserToken+" envvar.")
```